test/tsconfig.json

```diff
@@ -142,4 +143,4 @@
         "rule-tester/testData.ts",
         "rule-tester/utilsTests.ts"
     ]
-}
+}
```

Comment:
Oh, just ignore my previous comment in that case, it seems as if `grunt-ts` always removes this when rewriting `tsconfig.json` files. We might as well remove it and leave it removed.